pkg/operatorsource/configuring.go

```diff
@@ -95,20 +131,86 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *marketplace.O
 	}
 
 	// If we are here, the given CatalogSourceConfig object already exists.
-	cscNamespacedName := types.NamespacedName{Name: in.Name, Namespace: in.Namespace}
-	cscExisting := marketplace.CatalogSourceConfig{}
-	err = r.client.Get(ctx, cscNamespacedName, &cscExisting)
+	err = r.updateExistingCatalogSourceConfig(ctx, in, packages)
 	if err != nil {
-		r.logger.Errorf("Unexpected error while getting CatalogSourceConfig: %s", err.Error())
+		r.logger.Errorf("Unexpected error while updating CatalogSourceConfig: %s", err.Error())
 		nextPhase = phase.GetNextWithMessage(phase.Configuring, err.Error())
-
 		return
 	}
 
+	r.logger.Info("CatalogSourceConfig object has been updated successfully")
+
+	nextPhase = phase.GetNext(phase.Succeeded)
+	return
+}
+
+// getManifestMetadata gets the package metadata from the OperatorSource endpoint
+// it returns the list of packages to be written to the OperatorSource status and an error
+func (r *configuringReconciler) getManifestMetadata(spec *marketplace.OperatorSourceSpec, namespace string) ([]*datastore.RegistryMetadata, error) {
+
+	manifests := make([]*datastore.RegistryMetadata, 0)
```

Comment:
Like I mentioned before, this variable also needs to be renamed

Reply:
what do we want to call it? `metadata`? `manifestMetadata`?

Reply:
`metadata` works for me